client/js/components/assessments/item.jsx

```diff
@@ -7,211 +7,124 @@ import UniversalInput         from "./universal_input";
+                        Submit
+                      </button>
+                    </div>;
+    }
+    return submitButton;
+  }
+
+  getCounter(){
+    if(this.props.shouldShowCounter){
+      return <span className="counter">
+              {this.props.currentItemIndex + 1} of {this.props.questionCount}
+             </span>;
+    }
+    return;
+  }
 
   getResult(index){
     var result;
 
-    if(index == -1){
+    if(index == CORRECT_RESPONSE){
       result = <div className="check_answer_result">
-                <p></p>
-              </div>;
-    }
-    else if(index == 0){
-    result = <div className="check_answer_result">
-                <p>Incorrect</p>
-              </div>;
-    }
-    else {
-      result =  <div className="check_answer_result">
                   <p>Correct</p>
-                </div>;
+               </div>;
+    } else if(index == INCORRECT_RESPONSE) {
+      result = <div className="check_answer_result">
+                <p>Incorrect</p>
+               </div>;
+    } else {
+      result = <div className="check_answer_result">
+                <p></p>
+               </div>;
     }
 
     return result;
   }
 
 
   render() {
-    var unAnsweredWarning = this.getWarning(this.state, this.props.questionCount, this.props.currentItemIndex);
     var result = this.getResult(this.props.messageIndex);
-    var must_answer_message = this.state && this.state.showMessage ? <div>You must select an answer before continuing.</div> : "";
-    var confidenceButtons = this.getConfidenceLevels(this.props.confidenceLevels);
-    var submitButton = (this.props.currentItemIndex == this.props.questionCount - 1) ? <button className="btn btn-check-answer" onClick={(e)=>{this.submitButtonClicked(e)}}>Submit</button> : "";
-    var footer = this.getFooterNav();
-    var navigationDiv = this.getNavigationButtons();
-
-    //Check if we need to display the counter in the top right
-    var counter = "";
+    var navigation = this.getNavigationButtons();
 
-    if(this.props.shouldShowCenter){
-      counter = <span className="counter">{this.props.currentItemIndex + 1} of {this.props.questionCount}</span>
-    }
-    var formativeHeader = "";
-    // if(this.props.settings.assessmentKind.toUpperCase() == "FORMATIVE"){
-    //   formativeHeader =
-    //       <div>
-    //         <div className="row">
-    //         </div>
-    //         <div className="row">
-    //           <div className="col-md-10">
-    //             <h4>{this.props.assessment.title}</h4>
-    //           </div>
-    //           <div className="col-md-2">
-    //           </div>
-    //         </div>
-    //       </div>
-    // }
-
-    var submitButtonDiv =  <div>
-                          {submitButton}
-                        </div>;
-
-    // if(this.props.settings.assessmentKind.toUpperCase() == "FORMATIVE"){
-    //   submitButtonDiv = ""
-    // }
+    var counter = this.getCounter();
+    let submitButton = this.getSubmitButton();
```

Comment:
Make this a `var`.  It is equivalent to one in this position, and matches the other declarations.